### DISCUSSION TITLE
Why does my domain hwm.monster have an Invalid Configuration error?

### DISCUSSION THREAD

**Discussion by ardhichoiruddin** [vercel/community#4569] [Help]
### Summary

I have a hwm.monster domain and I integrated it with Vercel. for DNS I use Cloudflare. But why is there an "Invalid Configuration" error? Even though there is this error, my domain or website can still be accessed, but if the URL is shared on social media such as Facebook, the URL responds with "Invalid SSL Certificate".
in the vercel dashboard /username/your-website/settings/domains, sometimes the domain is "valid configuration" and sometimes "invalid configuration".

### Steps to Reproduce

Add A record in Cloudflare with value 76.76.21.21

**Comment by amyegan** [comment]
There are some tricks you can use when setting up a domain with Cloudflare: https://vercel.com/guides/using-cloudflare-with-vercel 

What is the domain?
